### DISCUSSION TITLE
Typo

### DISCUSSION THREAD

**Discussion by kgrigsby59** [pointfreeco/swift-composable-architecture#1372] [Beta]
https://github.com/pointfreeco/swift-composable-architecture/blob/d234dd5fb49e75eaf610c1ae770cf6ce4799f730/Sources/Dependencies/Dependencies/UUID.swift#L69
Should be:
///     Todo(id: UUID(uuidString: "00000000-0000-0000-0000-000000000000")!)

**Comment by stephencelis** [comment]
@kgrigsby59 Thanks for letting us know!

This isn't so much a discussion topic as much as it is a pull request or issue, though. If you visit this source code on the branch there should be an option to "edit this file" (a pencil icon I think) and pull request the change:

https://github.com/pointfreeco/swift-composable-architecture/blob/protocol-beta/Sources/Dependencies/Dependencies/UUID.swift#L69

If you have a chance, can you submit that instead? Thanks!

**Reply by stephencelis** [reply]
(If not we'll try to get to it soon, but if you see any other typos, please submit fixes via GitHub's edit function, or via issues, instead.)
